q2_qemistree/plugin_setup.py

```diff
@@ -224,4 +230,41 @@
                                         'tips that are in feature data'}
 )
 
+plugin.visualizers.register_function(
+    function=plot,
+    name='Generate an iTOL bar chart annotation file',
+    description=('Calculate mean frequency per category per sample and render '
+                 'as bar height.'),
+    inputs={'table': FeatureTable[Frequency],
+            'tree': Phylogeny[Rooted],
+            'feature_metadata': FeatureData[Molecules]
+            },
+    parameters={
+        'coloring_category': Str,
+        'parent_mz': Str,
+        'color_palette': Str % Choices(['Pastel1', 'Pastel2', 'Paired',
+                                        'Accent', 'Dark2', 'Set1', 'Set2',
+                                        'Set3', 'tab10', 'tab20', 'tab20b',
+                                        'tab20c', 'Greys', 'Purples', 'Blues',
+                                        'Greens', 'Oranges', 'Reds', 'YlOrBr',
+                                        'YlOrRd', 'OrRd', 'PuRd', 'RdPu',
+                                        'BuPu', 'GnBu', 'PuBu', 'YlGnBu',
+                                        'PuBuGn', 'BuGn', 'YlGn']),
+        'ms2_label': Bool,
+                },
+    input_descriptions={'table': 'Table of feature sample categories',
+                        'tree': 'Phenetic tree',
+                        'feature_metadata': 'Feature metadata'
+                        },
+    parameter_descriptions={
+        'coloring_category': 'The feature data column used to color the tips',
```

Comment:
In the current form, this column is used to color and label both

Reply:
Renamed